tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es6",
+    "target": "es2022",
```

Comment:
Why has this changed @mayura-andrew?

Reply:
for some testing purposes i'll remove this